README.md

```diff
@@ -6,6 +6,7 @@
 [![Twitter Follow](https://img.shields.io/twitter/follow/expectgreatdata?style=social)](https://twitter.com/expectgreatdata)
 [![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://greatexpectations.io/slack)
 [![Contributors](https://img.shields.io/github/contributors/great-expectations/great_expectations)](https://github.com/great-expectations/great_expectations/graphs/contributors)
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/charliermarsh/ruff/main/assets/badge/v1.json)](https://github.com/charliermarsh/ruff)
```

Comment:
❤️ love your attention to detail with these things

Reply:
just a shield in our README

Reply:
The most important part of the PR 😆 
![image](https://user-images.githubusercontent.com/13108583/214153700-4c339feb-6203-4a54-8896-2d0438039d11.png)